Showcase-iOS/Login/View/LoginVC.swift

```diff
@@ -10,8 +10,29 @@ import UIKit
 
 class LoginVC: UIViewController {
```

Comment:
The name of this class is really bad. Please can you rename it to : LoginViewController. Naming is important and we need to not take short cuts when naming things.